index.html

```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
+            </button>
+          </div>
+        </div>
+    </div>
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (3).png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
+            
+            <button type="submit" id="project-view">
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (4).png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
+            
+            <button type="submit" id="project-view">
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (5).png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
+            
+            <button type="submit" id="project-view">
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+  </section>
+  <section class="about">
+    <div class="about-me">
+        <h1>About</h1>
```

Comment:
- [ ] Let's ensure that the content of this About section is aligned using CSS flexbox 🗳️